src/Encoder/JsonldEncoder.php

```diff
@@ -23,6 +23,7 @@ class JsonldEncoder extends SymfonyJsonEncoder {
    * {@inheritdoc}
    */
   public function encode($data, $format, array $context = array()) {
```

Comment:
@ruebot to avoid the last issue with "override" not being used, removing from line 22 to line 30 should allow phpcs to pass. Thanks.